iocore/net/SSLSNIConfig.cc

```diff
@@ -93,6 +96,78 @@ NamedElement::set_regex_name(const std::string &regex_name)
 ////
 // SNIConfigParams
 //
+// ----
+
+SNI_IpAllow::SNI_IpAllow(std::string &ip_allow_list, std::string const &servername)
+{
+  ts::TextView content{ip_allow_list};
+  if (content && content[0] == '@') {
+    std::error_code ec;
+    ts::file::path path{content.remove_prefix(1)};
+    ts::LocalBufferWriter<1024> w;
+    if (path.is_relative()) {
+      path = ts::file::path(Layout::get()->sysconfdir) / path;
+    }
+    ip_allow_list = ts::file::load(path, ec);
+    if (ec) {
```

Comment:
This code allows the user to reference an external file that contains a list of IPs to filter on. Can you update the documentation with this functionality?

Reply:
updated the docs under the `ip_allow` configuration.